packages/gestalt/src/Table.css

```diff
@@ -14,6 +16,15 @@
   composes: paddingX3 from "./boxWhitespace.css";
 }
 
-.thead {
-  composes: borderBottom from "./Borders.css";
+.trClickable:hover {
+  background-color: #efefef;
+  cursor: pointer;
+}
+
+.trClickable:hover td:first-child {
```

Comment:
@PayalModi could we split the border radius properties up? I know it's a bit of an edge case but this would break if the table only had 1 column:
![image](https://user-images.githubusercontent.com/127199/82491928-3a2ead00-9a9a-11ea-9ecd-aebc672b0660.png)

```css
border-top-left-radius: 8px;
border-bottom-left-radius: 8px;
```